.babelrc

```diff
@@ -13,7 +13,12 @@
     // default in babel v6.
     ["@babel/plugin-proposal-class-properties", { loose: true }],
     ["@babel/plugin-proposal-object-rest-spread", { useBuiltIns: true }],
-    "syntax-dynamic-import"
+    "syntax-dynamic-import",
+    ["module-resolver", {
+      "alias": {
+        "firefox-profiler": "./src",
+      }
+    }]
```

Comment:
At first, I was very surprised that this works for tests too (normally for tests we use the environment below). I thought that the list of plugins was replaced. TIL I see they're actually _merged_ (see https://babeljs.io/docs/en/options#env and https://babeljs.io/docs/en/options#merging)... Which means that we could possibly simplify the test environment further!
(I'm pretty sure this wasn't the case when I did that at first with babel v6, possibly babel v7 changed this.)

So initially I wanted to suggest to move this to the test entry + configure webpack for the other environments, but then I found that this works perfectly like this, so let's keep it like this!